### PR TITLE
[RHSTOR-5136] Updates for building client operator UI

### PR DESCRIPTION
**Pull request by bipuladh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,13 +25,15 @@
     "build:plugin": "yarn build:generate && NODE_OPTIONS='--max-old-space-size=4096' yarn ts-node node_modules/.bin/webpack -c ./webpack.config.ts",
     "build": "NODE_ENV=production PLUGIN=odf I8N_NS=plugin__odf-console yarn build:plugin",
     "build-mco": "NODE_ENV=production PLUGIN=mco I8N_NS=plugin__odf-multicluster-console yarn build:plugin",
+    "build-client": "NODE_ENV=production PLUGIN=client I8N_NS=plugin__odf-multicluster-console yarn build:plugin",
```
Comment (SanjalKatiyar) [inline package.json:28]: `I8N_NS=plugin__odf-multicluster-console` ??

Comment (GowthamShanmugam) [inline package.json:28]: same here https://github.com/red-hat-storage/odf-console/pull/1120/files#r1418787638

**Review by GowthamShanmugam — APPROVED** [review]

**Comment by bipuladh** [comment]
/override ci/prow/odf-console-e2e-aws
As changes are unrelated to the CI  tests

**Pull request merged**